app/views/content_items/service_sign_in/_choose_sign_in.html.erb

```diff
@@ -1,12 +1,31 @@
 <%= render "components/back-link", href: @content_item.back_link %>
 
+<% if @error %>
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <%= render "components/error-summary", {
+        title: t('service_sign_in.error.title'),
+        items: [
+          {
+            text: t('service_sign_in.error.option'),
+            href: "#option-0"
+          }
+        ]
+      } %>
+    </div>
+  </div>
+<% end %>
+
 <%= form_tag({controller: 'content_items', action: 'service_sign_in_options'}, method: "post", data: { module: 'track-radio-group' }) do %>
   <% legend_text = render 'govuk_component/title', title: @content_item.title %>
   <%= render "components/fieldset", legend_text: legend_text do %>
     <div class="grid-row">
       <div class="column-two-thirds">
         <%= render 'govuk_component/govspeak', content: @content_item.description %>
-        <%= render "components/radio", name: "option", items: @content_item.options %>
+        <% if @error %>
+          <%= render "components/error-message", text: t('service_sign_in.error.option') %>
+        <% end %>
+        <%= render "components/radio", id_prefix: "option", name: "option", items: @content_item.options %>
```

Comment:
How is adding the id_prefix to the option related to the error component?

Reply:
Maybe we should have a test that makes sure these things are the same...